crates/uv-build/src/lib.rs

```diff
@@ -737,86 +736,159 @@ impl SourceBuild {
+                    .temp_dir
+                    .path()
+                    .join(format!("build_{}.txt", self.build_kind));
+
+                debug!(
+                    r#"Calling `{}.build_{}("{}", {}, {})`"#,
+                    pep517_backend.backend,
+                    self.build_kind,
+                    output_dir.escape_for_python(),
+                    self.config_settings.escape_for_python(),
+                    metadata_directory,
+                );
+                let script = formatdoc! {
+                    r#"
+                    {}
+
+                    wheel_filename = backend.build_{}("{}", {}, {})
+                    with open("{}", "w") as fp:
+                        fp.write(wheel_filename)
+                    "#,
+                    pep517_backend.backend_import(),
+                    self.build_kind,
+                    output_dir.escape_for_python(),
+                    self.config_settings.escape_for_python(),
+                    metadata_directory,
+                    outfile.escape_for_python()
+                };
+                let span = info_span!(
+                    "run_python_script",
+                    script=format!("build_{}", self.build_kind),
+                    python_version = %self.venv.interpreter().python_version()
+                );
+                let output = self
+                    .runner
+                    .run_script(
+                        &self.venv,
+                        &script,
+                        &self.source_tree,
+                        &self.environment_variables,
+                        &self.modified_path,
+                    )
+                    .instrument(span)
+                    .await?;
+                if !output.status.success() {
+                    return Err(Error::from_command_output(
+                        format!(
+                            "Build backend failed to build wheel through `build_{}()`",
+                            self.build_kind
+                        ),
+                        &output,
+                        &self.version_id,
+                    ));
+                }
 
-        let distribution_filename = fs::read_to_string(&outfile)?;
-        if !wheel_dir.join(&distribution_filename).is_file() {
-            return Err(Error::from_command_output(
-                format!(
-                    "Build backend failed to produce wheel through `build_{}()`: `{distribution_filename}` not found",
-                    self.build_kind
-                ),
-                &output,
-                &self.version_id,
-            ));
+                let distribution_filename = fs::read_to_string(&outfile)?;
+                if !output_dir.join(&distribution_filename).is_file() {
+                    return Err(Error::from_command_output(
+                        format!(
+                            "Build backend failed to produce through `build_{}()`: `{distribution_filename}` not found",
```

Comment:
Word missing; Can we use `self.build_kind` and also move this out of the match?

Reply:
I thought we wanted it to show `wheel` when it's editable, but otherwise yes.